src/common.toit

```diff
@@ -103,6 +207,33 @@ abstract class Canvas:
       --source_line_stride/int   // In bytes.
       --orientation/int
 
+  /**
+  Draws the given 8-bit pixmap on the canvas.
+  The source pixmap has one byte per pixel, which is an index into the
+    $palette and $alpha arguments.  The order of the pixmap is as in
+    PNG, so the lines are ordered from top to bottom, and within each
+    line the bytes are ordered from left to right.
+  The $alpha argument controls which pixel indexes are transparent.  A
+    byte value of 0 means pixels with that index are transparent, and a byte
+    value of 0xff means the pixels with that index are opaque.
```

Comment:
Does the alpha bytearray need to be 256 bytes long? Do all colors need to have an entry?

Reply:
This is answered in the line below.